SETUP/tests/unittests/ParamValidatorTest.php

```diff
@@ -309,6 +320,78 @@ public function testFloatMoreThanMax()
         get_float_param($this->GET, 'f10', $default, $min, $max);
     }
 
+    //------------------------------------------------------------------------
+    // get_bool_param() tests
+
+    public function testBool()
```

Comment:
This test is identical to the second test case in `testBoolTrueVariants`, is it worth the duplication?

Reply:
Yup, removed.